projects/systelab-components/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "systelab-components",
-  "version": "10.0.15",
+  "version": "10.3.0",
```

Comment:
10.1.0 was already published last week
10.2.0 will be published from branch 10.2.x
We update to 10.3.0 for future releases